share/mk/bsd.compat.mk

```diff
@@ -157,9 +157,13 @@ LIB64_MACHINE_ABI=	${MACHINE_ABI:Npurecap}
 .endif
 HAS_COMPAT+=CHERI
 LIBCHERICPUFLAGS=  -target mips64-unknown-freebsd13.0 -cheri -mabi=purecap
+LIBCHERICPUFLAGS+=	-fpic
+LIBCHERICPUFLAGS+=	-Qunused-arguments
+LIBCHERICPUFLAGS+=	-Werror=cheri-bitwise-operations
 LIBCHERI_MACHINE=	mips
 LIBCHERI_MACHINE_ARCH=	mips64c128
-
+LIBCHERILDFLAGS=	-fuse-ld=lld
+LIBCHERILDFLAGS+=	-Wl,-preemptible-caprelocs=elf
```

Comment:
Should be the default so we can drop this.

Reply:
Done (in both places)